react/src/App.js

```diff
@@ -1,30 +1,21 @@
 import React, { Component } from 'react';
-import BookList from './components/BookList';
-import BookFilters from './components/BookFilters';
-import filterFieldConstants from './constants/filterFieldConstants'
+import {
+  BrowserRouter as Router,
+  Route
+} from 'react-router-dom';
 import './styles.css';
 
-class App extends Component {
-  state = { filterField: filterFieldConstants.defaultFilterField, filterText: '' };
-
-  handleTextFilter = (filterText) => this.setState({ filterText });
-  handleFieldFilter = (filterField) => this.setState({ filterField });
+import BookSearch from './screens/BookSearch';
 
+class App extends Component {
   render() {
     return (
-      <div className="book-search">
-        <BookFilters
-          filterField={this.state.filterField}
-          filterText={this.state.filterText}
-          onFilterTextChange={this.handleTextFilter}
-          onFilterFieldChange={this.handleFieldFilter}
-        />
-        <BookList
-          filterField={this.state.filterField}
-          filterText={this.state.filterText}
-        />
-      </div>
-    );
+      <Router>
+        <div>
```

Comment:
Do we need this `div` ?

Reply:
What we usually do is to put a `Switch` inside the `Router` and then, the `Routes` inside of that `Switch`. Instead of removing this `div` I'd change this to a `Switch`. What do you think ?